modules/deployment/src/test/java/org/mule/runtime/module/deployment/internal/ApplicationDeploymentTestCase.java

```diff
@@ -813,6 +814,158 @@ public void undeploysNotStoppedAppAndStartsItOnDeploy() throws Exception {
     assertStatus(app_2, STARTED);
   }
 
+  @Issue("MULE-19127")
```

Comment:
I would try to reuse code in this set of tests if possible (it is performing the same steps, it varies whether the flows are/are not stopped). Maybe it would be preferable to group some steps (stopTestFlows, restartingApps, assertX, etc.)